pages/glossary/distinguishable-text-styles.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Distinguishable text styles
+key: distinguishable-text-styles
+unambiguous: true
+objective: true
+---
+
+Two elements have _distinguishable text styles_ if they have different [computed values][] for at least one [text style property](#text-style-properties), unless one element is [descendant][] of the other and differences are not only due to the [descendant][] element having any of the following [computed values][]:
```

Comment:
Elements can have different styles, even if their text content doesn't. Consider the following example:

```html
<span id="elm1" style="font-weight: 300">Hello</span>
<strong id="elm2"><span style="font-weight: 300">world</span></strong>
```

This definition needs to account for the style of the parent of a text node, not jut any two elements.

Reply:
I don't think I agree. This definition compares text styles of 2 elements. Deciding which elements get compared is responsibility of the rule, not of the definition.